theme/fieldeditors.less

```diff
@@ -143,6 +143,8 @@
 
 .blocklyWidgetDiv .blocklyGridPickerMenu .goog-menuitem-content {
     color: #fff;
+    font-size: 13px;
```

Comment:
13px seems quite small. Does it pass a11y testing?

Reply:
none of our blocks pass a11y testing, sadly! this is what we have in live